storage/db-tool/src/gen_replay_verify_jobs.rs

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use aptos_backup_cli::{
+    metadata::{
+        cache::{sync_and_load, MetadataCacheOpt},
+        StateSnapshotBackupMeta,
+    },
+    storage::DBToolStorageOpt,
+    utils::ConcurrentDownloadsOpt,
+};
+use aptos_logger::warn;
+use aptos_types::transaction::Version;
+use clap::Parser;
+use itertools::Itertools;
+use std::{io::Write, iter::once, path::PathBuf};
+
+#[derive(Parser)]
+pub struct Opt {
+    #[clap(flatten)]
+    metadata_cache_opt: MetadataCacheOpt,
+    #[clap(flatten)]
+    storage: DBToolStorageOpt,
+    #[clap(flatten)]
+    concurrent_downloads: ConcurrentDownloadsOpt,
+    #[clap(
+        long,
+        help = "The first transaction version required to be replayed and verified. [Defaults to 0]"
+    )]
+    start_version: Option<Version>,
+    #[clap(
+        long,
+        help = "Target number of transactions for each job to replay",
+        default_value = "20000000"
```

Comment:
nit - leave a brief comment why this value is used. like it takes x hours to run.

Reply:
will do after I finished experimenting on the values